bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -337,7 +336,16 @@ public int getMaxScanTimeoutRetries() {
   public boolean isRetryable(Status.Code code) {
     return statusToRetryOn.contains(code);
   }
-  
+
+  /**
+   * Should returns {@link Set} of {@link Status.Code} based on which RPC retries.
```

Comment:
I think this would be better: `Returns {@link Set} of {@link Status.Code} that allow RPC retries.`

Reply:
Thanks for the input. Updated Javadoc with the same.